libs/barista-components/event-chart/src/event-chart.scss

```
@@ -162,6 +162,8 @@
 }
 
 /** Event chart overlay */
-::ng-deep .dt-event-chart-overlay-panel {
-  @include dt-overlay-container();
+::ng-deep .cdk-overlay-pane {
+  &.dt-event-chart-overlay-panel {
+    @include dt-overlay-container();
+  }
```

```suggestion
/** The selector is combined with the cdk-overlay-pane to increase the specificity to avoid edge cases where the cdk styles are loaded after the component's styles in the angular.json */
::ng-deep .cdk-overlay-pane.dt-event-chart-overlay-panel  {
    @include dt-overlay-container();
```
No need for nesting here - it makes it harder to read and parse selectors for developers :) Please see our coding guidelines https://github.com/dynatrace-oss/barista/blob/master/CODING_STANDARDS.md#scss-specifics